src/allotropy/parsers/biorad_bioplex/biorad_bioplex_structure.py

```diff
@@ -0,0 +1,261 @@
+from dataclasses import dataclass, field
+
+from defusedxml import ElementTree as DefusedElementTree
+
+from allotropy.exceptions import AllotropeConversionError
+from allotropy.parsers.utils.values import get_val_from_xml, try_int_or_none
+
+WELLS_TAG = "Wells"
+DOC_LOCATION_TAG = "NativeDocumentLocation"
+DESCRIPTION_TAG = "Description"
+PLATE_DIMENSIONS_TAG = "PlateDimensions"
+TOTAL_WELLS_ATTRIB = "TotalWells"
+VERSION_ATTRIB = "BioPlexManagerVersion"
+SAMPLES = "Samples"
+MEMBER_WELLS = "MemberWells"
+ROW_NUMBER = "RowNo"
+COLUMN_NUMBER = "ColNo"
+WELL_NUMBER = "WellNumber"
+WELL_NO = "WellNo"
+REGION_NUMBER = "RegionNumber"
+REGIONS_OF_INTEREST = "RegionsOfInterest"
+CODE = "Code"
+DILUTION = "Dilution"
+ROW_NAMES = "ABCDEFGH"
+
+SAMPLE_ROLE_TYPE_MAPPING = {
+    "Blank": "blank role",
+    "Control": "control sample role",
+    "Standard": "standard sample role",
+    "Unknown": "unknown sample role",
+}
+
+
+@dataclass(frozen=True)
+class AnalyteSample:
+    analyte_name: str
+    analyte_region: int
+    analyte_error_code: int
+
+    @staticmethod
+    def create(analyte_xml):
```

Comment:
Note: adding a return definition of "AnalyteSample", results in this error: NameError: name 'AnalyteSample' is not defined